src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,56 @@
+
+
+
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      printTimeCallback && printTimeCallback();
+    },10)
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 6000);
   }
 
   getSeconds() {
-    // ... your code goes here
+   return Math.floor((this.currentTime / 100) % 60);
   }
 
   getCentiseconds() {
-    // ... your code goes here
-  }
+    return this.currentTime % 100;
+  } 
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    let digit = value.toString();
```

Comment:
Salvo que reasigneis el valor, como haceis con el result despues mejor siempre poner const!
```suggestion
    const digit = value.toString();
```